README.md

```diff
@@ -25,13 +25,15 @@ RAILS_ENV=production bin/rolling_index stop
 
 See https://sul-dlss.github.io/dor_indexing_app/
 
-## Setup RabbitMQ
-You must set up the durable rabbitmq queues that bind to the exchange where workflow messages are published.
+## RabbitMQ Setup
+
+NOTE: This is done automatically by Capistrano on deployment, so one generally shouldn't need to run this task manually. This is for informational purposes only.
```

Comment:
I don't think we want to do this on every deploy.  We only want to run this when the bindings change.

Reply:
@jcoyne Can you say why we might not want to do this every time? It seems pretty cheap. Maybe it has side effects we don't want?

Reply:
@jcoyne I'm going to noodle on this a bit and see if I can't get a quick improvement in. Stay tuned.

Reply:
@jcoyne meh, not sure it's worth any more noodling time. I'm still interested in your answer to what I asked above, tho.

Reply:
Deploying is something we do several times a week, but I don't anticipate we need to add any further bindings.  Each step in the deploy adds a few seconds and an extra point where problems can arise. I think having small special purpose tools makes things work better and easier to understand.